CRM/Import/DataSource.php

```diff
@@ -34,14 +34,26 @@ abstract class CRM_Import_DataSource implements DataSourceInterface {
    */
   private $limit;
 
+  public function getLimit(): int {
```

Comment:
If `$limit` is not initialized it will be `null` so based on reading the code I think this should be 
```suggestion
  public function getLimit(): ?int {
```

Reply:
hmm - good point - I will check on that

Reply:
I guess the reason I haven't hit this in practice is that it is only `got` when it would be set - it is not not-set in the cases where you would look at it - which makes me wonder if it's 'right' for it to error in that case

Reply:
OK - so this is where it is retrieved

![image](https://github.com/civicrm/civicrm-core/assets/336308/947bd02e-b494-429b-80d6-92f0a2fa4c97)


I think it's not really valid to call `getLimit()` before `setLimit()` is set - so this might be right

Reply:
Ok fair enough.

Reply:
@colemanw I guess I lean towards strict here because there are no existing users of the function - so easier to loosen later than to tighten